settings/LoanPolicyDetail.js

```diff
@@ -153,15 +173,19 @@ class LoanPolicyDetail extends React.Component {
           </Col>
         </Row>
         <br />
-        <Row>
-          <Col xs={12}>
-            <KeyValue
-              label={<FormattedMessage id="ui-circulation.settings.loanPolicy.openingTimeOffset" />}
-              value={`${get(policy, ['loansPolicy', 'openingTimeOffset', 'duration'], '')} ${timeOffsetInterval}`}
-            />
-          </Col>
-        </Row>
-        <br />
+        {(isOpeningTimeOffsetVisible()) &&
```

Comment:
`this.isOpeningTimeOffsetVisible()` and I suggest to move it to a separate variable